store/filecache.go

```diff
@@ -1,7 +1,6 @@
 package store
```

Comment:
Probably. I originally wrote filecache as a wrapper for remote store but it morphed substantially since then.